### ISSUE TITLE
[Bug]: Cannot find manifest in repo

### ISSUE THREAD

**Issue by Ubujira** [russellbanks/Komac#446]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Manifest `mcmilk.7zip-zstd` cannot be found in the `winget-pkgs` repository despite [existing](https://github.com/microsoft/winget-pkgs/tree/master/manifests/m/mcmilk/7zip-zstd).

**Command**
`
komac update -i mcmilk.7zip-zstd -v "22.01-ZS-v1.5.5-R3" -u https://github.com/mcmilk/7-Zip-zstd/releases/download/v22.01-v1.5.5-R3/7z22.01-zstd-x64.exe https://github.com/mcmilk/7-Zip-zstd/releases/download/v22.01-v1.5.5-R3/7z22.01-zstd-x32.exe https://github.com/mcmilk/7-Zip-zstd/releases/download/v22.01-v1.5.5-R3/7z22.01-zstd-arm64.exe https://github.com/mcmilk/7-Zip-zstd/releases/download/v22.01-v1.5.5-R3/7z22.01-zstd-arm.exe
`

**Ouput**
```
Error:
   0: mcmilk.7zip-zstd does not exist in microsoft/winget-pkgs
   1: No files were found for manifests/m/mcmilk/7zip-zstd
   ```

**Comment by russellbanks** [comment]
The cause of this is the same as #445 as Komac retrieves those versions fine, but filters out ones that aren't valid, presumably because they all are. This is an issue in the library Komac uses for parsing and sorting versions, [fosskers/rs-versions](https://github.com/fosskers/rs-versions), which I've reported.

**Comment by russellbanks** [comment]
Closing as duplicate as the cause is the same as #447 and will be fixed once #447 is resolved.
